Tests/SwiftAlgorithmsTests/LazySplitCollectionTests.swift

```diff
@@ -0,0 +1,218 @@
+//===----------------------------------------------------------------------===//
+//
+// This source file is part of the Swift Algorithms open source project
+//
+// Copyright (c) 2021 Apple Inc. and the Swift project authors
+// Licensed under Apache License v2.0 with Runtime Library Exception
+//
+// See https://swift.org/LICENSE.txt for license information
+//
+//===----------------------------------------------------------------------===//
+
+import XCTest
+
+@testable import Algorithms
```

Comment:
Do we really need testable import here? For what I could see we only test the public API so we should be fine without it
```suggestion
import Algorithms
```

Reply:
You found a serious error! I hadn't actually made the new `split`s I'm adding public. I've learned a valuable lesson: `@testable` is a big hammer that can hide access control bugs. Thank you.